src/saveload/saveload.h

```diff
@@ -407,6 +407,7 @@ enum SaveLoadVersion : uint16_t {
 	SLV_INDUSTRY_NUM_VALID_HISTORY,         ///< 356  PR#14416 Store number of valid history records for industries.
 	SLV_INDUSTRY_ACCEPTED_HISTORY,          ///< 357  PR#14321 Add per-industry history of cargo delivered and waiting.
 	SLV_TOWN_SUPPLY_HISTORY,                ///< 358  PR#14461 Town supply history.
+	SLV_STATIONS_UNDER_BRIDGES,             ///< 359  PR#14477 Allow stations under bridges.
```

Comment:
This savegame version doesn't seem to be used anywhere in the PR. Am I missing it?

Reply:
No conversion is necessary. The savegame bump is to prevent loading a savegame with bridges over stations into a version of the game that doesn't support it.